src/core_atmosphere/physics/mpas_atmphys_control.F

```diff
@@ -368,6 +374,19 @@ subroutine physics_registry_init(mesh,configs,sfc_input)
              dzs(4,iCell) = 1.00_RKIND
           enddo
 
+       case("sf_ruc")
+         do iCell = 1,nCells
+             dzs(1,iCell) = 0.005_RKIND
+             dzs(2,iCell) = 0.025_RKIND
+             dzs(3,iCell) = 0.045_RKIND
+             dzs(4,iCell) = 0.13_RKIND
+             dzs(5,iCell) = 0.25_RKIND
+             dzs(6,iCell) = 0.35_RKIND
+             dzs(7,iCell) = 0.50_RKIND
+             dzs(8,iCell) = 1.00_RKIND
+             dzs(9,iCell) = 0.70_RKIND
```

Comment:
@tanyasmirnova : is this really correct? the 9th layer is thinner than the 8th?

Reply:
@joeolson42 Joe, these are the thicknesses between the middles of the layers. The first and last values are actually 0.5 of the thickness.

Reply:
OK. Thanks.